sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/LeaseStoreManager.java

```diff
@@ -52,6 +53,16 @@ interface LeaseStoreManagerBuilderDefinition {
      */
     Mono<Lease> createLeaseIfNotExist(String leaseToken, String continuationToken);
 
+    /**
+     * Checks whether the lease exists and creates it if it does not exist.
+     *
+     * @param leaseToken the partition to work on.
+     * @param continuationToken the continuation token if it exists.
+     * @param properties the properties.
```

Comment:
what is expected to go into these "properties"?

Reply:
To be frank, not sure what goes into the properties, as does not seem to really any values in it. 
But based on the current logic, the child leases get the properties from parent when the instance get direct child lease assignment during split.